source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst

```diff
@@ -0,0 +1,120 @@
+
+    ::
+
+        apt install nbdkit
+
+
+Supported Distributions for KVM Hypervisor:
+
+
+.. cssclass:: table-striped table-bordered table-hover
+
+========================    ========================
+Linux Distribution          Supported Versions
+========================    ========================
+Alma Linux                  8, 9
+Red Hat Enterprise Linux    8, 9
+Rocky Linux                 8, 9
+Ubuntu                      22.04 LTS
+========================    ========================
+
+
+Importing Windows guest VMs from VMware requires installing the virtio drivers on the hypervisor hosts for the virt-v2v conversion.
+
+On (RH)EL hosts:
+
+    ::
+
+        yum install virtio-win
+
+You can also install the RPM manually from https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.noarch.rpm
+
+
+For Debian-based distributions:
+
+    ::
+
+        apt install virtio-win
+
+Usage
+-----
+
+In the UI, Virtual Machines to import from VMware are listed in *Tools > Import-Export Instances* section, selecting:
+
+.. cssclass:: table-striped table-bordered table-hover
+
+==================== ========================
+Source               Destination Hypervisor  
+==================== ========================
+Migrate From VMware  KVM
+==================== ========================
+
+|import-vm-from-vmware-to-kvm.png|
+
+Selecting the Destination cluster
+---------------------------------
+
+CloudStack administrators must select a KVM cluster to import the VMware Virtual Machines (left side of the image above). Once a KVM cluster is selected, the VMware Datacenter selection part is displayed (right side of the image above).
+
+Selecting the VM from a VMware Datacenter
+-----------------------------------------
+
+CloudStack administrators must select the Source VMware Datacenter:
+
+    - Existing: The existing zones are listed, and for each zone CloudStack will list if there is any VMware Datacenter associated to it. In case it is, it can be selected
+    - External: CloudStack allows listing Virtual Machines from a VMware Datacenter that is not associated to any CloudStack zone. To do so, it needs the vCenter IP address, the datacenter name, and username and password credentials to log in the vCenter.
+
+Once the VMware Datacenter is selected, click on List VMware Instances to display the list of Virtual Machines on the Datacenter
+
+
+Converting and importing a VMware VM
+------------------------------------
+
+.. note:: CloudStack allows importing Running Linux Virtual Machines, but it is recommended that the Virtual Machine to import is powered off and has been gracefully shutdown before the process starts. For Windows Virtual Machines, it is not possible to import them while running. In case a Linux VM is imported while running, it will be converted in "crash consistent" state.
```

Comment:
```suggestion
.. note:: CloudStack allows importing Running Linux Virtual Machines, but it is generally recommended that the Virtual Machine to import is powered off and has been gracefully shutdown before the process starts. In case a Linux VM is imported while running, it will be converted in "crash consistent" state. For Windows Virtual Machines, it is not possible to import them while running, it is mandatory they are shut down gracefully so the filesystem is in a clean state.
```